packages/client/lib/sync/fullsync.ts

```diff
@@ -8,6 +8,12 @@ import { VMExecution } from './execution/vmexecution'
 import { TxPool } from './txpool'
 import { Event } from '../types'
 
+interface handledObject {
```

Comment:
Please keep interface and type names capitalized as per convention (see e.g. [styleguide here](https://github.com/basarat/typescript-book/blob/master/docs/styleguide/styleguide.md#interface)).

Will do these last updates on this locally and then push for review.